rest-api/dao/questionnaire_response_dao.py

```diff
@@ -241,6 +244,15 @@ def _update_participant_summary(
             new_status = QuestionnaireStatus.SUBMITTED_NO_CONSENT
           elif code.value == CONSENT_FOR_DVEHR_MODULE:
             new_status = dvehr_consent
+          elif code.value == CONSENT_FOR_STUDY_ENROLLMENT_MODULE:
+            # set language of consent to participant summary
+            for extension in resource_json.get('extension', []):
+              if extension['url'] == _LANGUAGE_EXTENSION and \
+                extension['valueCode'] in LANGUAGE_OF_CONSENT:
+                if participant_summary.primaryLanguage != extension['valueCode']:
+                  participant_summary.primaryLanguage = extension['valueCode']
+                  something_changed = True
+                break
```

Comment:
Are you sure you want a `break` here ? It will exit the entire for loop. Not sure what all resource_json could contain at this point. It may be right or you may want `continue`.

Reply:
Yes, it's break here. It just needs the first matched value.